### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by roshan-bajgain**
This is my portfolio first page.
The objective of the page is to use linter, github flow, documentation of repo.
I created My-portfolio repo and cloned into local machine.
I added HTML and css file and extract background image and some logo and implemented those tools

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @roshan-bajgain, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My-Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navbar">
+      <h1>My Logo</h1>
```
Comment (mke2111) [inline index.html:12]: - [ ] Enclose the logo within `a` tags. This will help to always redirect to a desired webpage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <img class="img1" src="Union.png" alt="Image of vector">
+    </nav>
+    <section>
+      <img class="img2" src="header.png" alt="Header-shaped mobile">
+      <header>
+        <P class="para">I’m Roshan <br> Glad to see<br> you!</P>
```
Comment (mke2111) [inline index.html:18]: - [ ] I recommend you to use `hx` tags for all titles. This is a title/heading so change the `p` tag to a `hx` tag, preferebly `h1` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <p class="para1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h2>LET’S CONNECT</h2>
+        <div class="img3">
+          <img class="img4" src="Vector1.png" alt="logo of social media">
```
Comment (mke2111) [inline index.html:22]: - [ ] For all these icons, group them using a `ul` tag. 
- [ ] For each icon, list it in an `li` tag
- [ ] Still, within the `li` tag, use `a` tag for each icon.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          <img src="Vector2.png" alt="logo of social media">
+          <img src="Vector3.png" alt="logo of social media">
+          <img class="img5" src="Vector4.png" alt="logo of social media">
+          <img class="img6" src="Vector5.png" alt="logo of social media">
+          <img class="img7" src="Vector6.png" alt="logo of social media">
+        </div>
+    </header>
+    </section>
+</body>
+</html>
```
Comment (mke2111) [inline index.html:32]: - [ ] Please consider the design template specifications used by figma. Your design is not responsive for all mobile screens. Adjust your design layout and positions appropriately.
Screenshot from 2021-08-18 13-59-12

File: index.html (hunk 1 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 2 above)
Comment (Emmyn5600) [reply index.html]: Well done

File: index.html (hunk 3 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 4 above)
Comment (Emmyn5600) [reply index.html:32]: Well done

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @roshan-bajgain 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
Optional suggestion

#### To remove unnecessary files in the root folder, you can create a new folder called `assets` to keep all images 

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**